{
  "uid" : "8c71822cb360f8c2",
  "name" : "POST Create user get to http",
  "fullName" : "tests.create_users_test.TestCreateUser#test_create_users_to_http",
  "historyId" : "07f87128a6eeb419bf537f16f4512fc8",
  "time" : {
    "start" : 1708412186983,
    "stop" : 1708412187048,
    "duration" : 65
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 200. Actual: 200",
  "statusTrace" : "self = <tests.create_users_test.TestCreateUser object at 0x7f66be605690>\n\n    @pytest.mark.xfail(reason='Bag')\n    @allure.title(\"POST Create user get to http\")\n    def test_create_users_to_http(self):\n        response = requests.post(url=f'{self.constant.WRONG_URL}api/users', headers=self.data.CREATE_USER)\n>       Assertions.assert_is_not_code_status(response, self.status.STATUS_CODE_200)\n\ntests/create_users_test.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, expected_status_code = 200\n\n    @staticmethod\n    def assert_is_not_code_status(response: Response, expected_status_code):\n        \"\"\"Check status code is wrong\"\"\"\n        actual_status_code = response.status_code\n        with allure.step(f'Expected status is not equal {expected_status_code}'):\n>           assert actual_status_code != expected_status_code, \\\n                f'Unexpected status code. Expected not {expected_status_code}. Actual: {actual_status_code}'\nE           AssertionError: Unexpected status code. Expected not 200. Actual: 200\n\nbase/assertions.py:27: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1708412183451,
      "stop" : 1708412183467,
      "duration" : 16
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "skipped",
    "statusMessage" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 200. Actual: 200",
    "statusTrace" : "self = <tests.create_users_test.TestCreateUser object at 0x7f66be605690>\n\n    @pytest.mark.xfail(reason='Bag')\n    @allure.title(\"POST Create user get to http\")\n    def test_create_users_to_http(self):\n        response = requests.post(url=f'{self.constant.WRONG_URL}api/users', headers=self.data.CREATE_USER)\n>       Assertions.assert_is_not_code_status(response, self.status.STATUS_CODE_200)\n\ntests/create_users_test.py:87: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, expected_status_code = 200\n\n    @staticmethod\n    def assert_is_not_code_status(response: Response, expected_status_code):\n        \"\"\"Check status code is wrong\"\"\"\n        actual_status_code = response.status_code\n        with allure.step(f'Expected status is not equal {expected_status_code}'):\n>           assert actual_status_code != expected_status_code, \\\n                f'Unexpected status code. Expected not {expected_status_code}. Actual: {actual_status_code}'\nE           AssertionError: Unexpected status code. Expected not 200. Actual: 200\n\nbase/assertions.py:27: AssertionError",
    "steps" : [ {
      "name" : "Expected status is not equal 200",
      "time" : {
        "start" : 1708412187048,
        "stop" : 1708412187048,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Unexpected status code. Expected not 200. Actual: 200\n",
      "statusTrace" : "  File \"/home/runner/work/reqres/reqres/base/assertions.py\", line 27, in assert_is_not_code_status\n    assert actual_status_code != expected_status_code, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Create User"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.xfail(reason='Bag')"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "create_users_test"
  }, {
    "name" : "subSuite",
    "value" : "TestCreateUser"
  }, {
    "name" : "host",
    "value" : "fv-az1116-403"
  }, {
    "name" : "thread",
    "value" : "2211-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.create_users_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 2,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "ae006c1e2fbb5a9",
        "reportUrl" : "https://Nat754.github.io/reqres/5//#testresult/ae006c1e2fbb5a9",
        "status" : "skipped",
        "statusDetails" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 200. Actual: 200",
        "time" : {
          "start" : 1708388624242,
          "stop" : 1708388624307,
          "duration" : 65
        }
      } ]
    },
    "tags" : [ "@pytest.mark.xfail(reason='Bag')" ]
  },
  "source" : "8c71822cb360f8c2.json",
  "parameterValues" : [ ]
}